Scripts/pilconvert.py

```diff
@@ -46,7 +46,7 @@ def usage():
     print(v)
     sys.exit(1)
 
-format = None
+outputFormat = None
```

Comment:
`output_format` instead of `outputFormat`

https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables